src/worker/api/index.ts

```diff
@@ -91,6 +94,24 @@ import {
 } from "../core/season/awards";
 import { getScore } from "../core/player/checkJerseyNumberRetirement";
 import type { NewLeagueTeam } from "../../ui/views/NewLeague/types";
+import * as filter_functions from "../core/trade/filterFunctions";
+
+export type OfferType = {
```

Comment:
I think you could save having to manually write out this type with something like `type OfferType = ThenArg<ReturnType<typeof augmentOffers>>;` Also it should be called `Offer` or `AugmentedOffer` rather than `OfferType` - we already know it's a type :)

Reply:
yeah, still pretty new to TS so I'm still winging it a bit here and there hah-- will fix

Reply:
I believe we can't reference augmentOffers, since it's scoped in getTradingBlockOffers. Let me know if you think we should extract that function